packages/request-client.js/src/api/request.ts

```diff
@@ -26,6 +26,8 @@ export default class Request {
   private contentDataExtension: ContentDataExtension | null;
   private emitter: EventEmitter;
 
+  private confirmationErrorOccurred: boolean = false;
```

Comment:
Could be explained here why this variable is used

Reply:
documented and renamed